hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/BackgroundService.java

```diff
@@ -141,7 +141,7 @@ public synchronized void run() {
             if (LOG.isDebugEnabled()) {
               LOG.debug("task execution result size {}", result.getSize());
             }
-          } catch (Exception e) {
+          } catch (Throwable e) {
```

Comment:
Thanks @ChenSammi for the this improvement. Can we also change the below `LOG` from `Warn` to `Error` ?